servers/src/main/java/tachyon/master/rawtable/RawTableMaster.java

```diff
@@ -138,7 +138,10 @@ public long createRawTable(TachyonURI path, int columns, ByteBuffer metadata)
 
     // Create a directory at path to hold the columns
     MkdirOptions options =
-        new MkdirOptions.Builder(MasterContext.getConf()).setRecursive(true).build();
+        new MkdirOptions.Builder(MasterContext.getConf())
+            .setPersisted(true)
```

Comment:
Original code will detect whether persist is true through `WriteType`. But I think that's fine since what we persist here are just directories. I was just curious about the reasonings behind, and I think you wanted to keep usages of mkdir consistent, sounds good to me.